WalletWasabi/Announcements/ReleaseHighlights.md

```diff
@@ -1,49 +1,38 @@
 ## Release Highlights
-#### 🤫 Send to Silent Payment addresses<br/>
-#### 🎁 Donation Button<br/>
-#### 🧪 ~~TestNet 3~~ -> TestNet 4<br/>
-#### 📝 Release Notes in the client<br/>
+#### 💪 3rd Party Providers for Fee & Exchange Rate<br/>
+#### 👨‍🔧 Quality of Life Features<br/>
+#### 📦 Backend and Coordinator packaged for Linux<br/>
 
 ## Release Summary
-Wasabi Wallet v2.4.0 brings send to Silent Payment addresses!
+Wasabi Wallet v2.5.0 continues to enhance Wasabi's resilience and improve the user interface to reduce frustration.
```

Comment:
` improve the user interface to reduce frustration.`
this is subjective, I'd rephrase/remove

Reply:
I dislike using this kind of acronyms UI/UX because many users won't know what they mean. I pushed a different solution